Library/Homebrew/cask/audit.rb

```diff
@@ -270,6 +273,83 @@ def bad_osdn_url?
       bad_url_format?(/osd/, [%r{\Ahttps?://([^/]+.)?dl\.osdn\.jp/}])
     end
 
+    def extract_url(url)
+      url.to_s.gsub(%r{.*"([a-z0-9]+://[^"]+)".*}m, '\1')
```

Comment:
I don't think this method is necessary since we don't store the URL with quotes.

Reply:
Sure.